apps/backend/src/transaction/interface/sbc_garms.ts

```diff
@@ -1,24 +1,65 @@
-export interface IGarmsPayment {
+export interface SBCGarmsRevenueAccount {
+  dist_client_code: string;
```

Comment:
SBCGarms to indicate that this is SBC specific and to differentiate between these interfaces and our other Transaction Interface